File: freecodecamp/daily-code-challenge/2026-03-19.py
def invert_matrix(matrix):
    # Get unique values
    values = set()
    for row in matrix:
        for val in row:
            values.add(val)

    a, b = list(values)

    # Create new matrix with swapped values
    return [[b if val == a else a for val in row] for row in matrix]

File: freecodecamp/daily-code-challenge/2026-03-19.js
function invertMatrix(matrix) {
  // Get the two unique values
  const values = new Set();
  for (let row of matrix) {
    for (let val of row) {
      values.add(val);
    }
  }

  const [a, b] = [...values];

  // Create new matrix with swapped values
  return matrix.map(row =>
    row.map(val => (val === a ? b : a))
  );
}
